lib/util/option.js

```diff
@@ -38,28 +38,35 @@ class Option {
 }
 
 class Some extends Option {
-  #value;
+  // Should.js uses `for ... in` to compare objects, so while should.js is in
+  // use an Option's value property must be "enumerable".
+  //
+  // See: https://github.com/shouldjs/equal/blob/master/index.js
+  // See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Enumerability_and_ownership_of_properties
+  //
+  // TODO revert from __please_use_get to #value when should.js is removed.
```

Comment:
I doubt it's an issue just with Should.js. I don't think any assertion library will be able to access a private property. It looks like Chai does something similar to Should.js: https://github.com/chaijs/deep-eql